src/test/java/com/epam/tc/hw3/testcases/Exercise1.java

```diff
@@ -0,0 +1,170 @@
+package com.epam.tc.hw3.testcases;
+
+import com.epam.tc.hw3.library.pages.Header;
+import com.epam.tc.hw3.library.pages.HomePage;
+import com.epam.tc.hw3.library.pages.SideBar;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.assertj.core.api.SoftAssertions;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+public class Exercise1 extends BaseTest {
+
+
+    //1. Open test site by URL - in parent class
+
+    @Test(priority = 0)
+    public void assertTitle() {
+        //2. Assert Browser title
+        SoftAssertions softly = new SoftAssertions();
+        String browserTitle = testDriver.getTitle();
+        softly.assertThat(browserTitle).as("Incorrect browser title").isEqualTo("Home Page");
+        softly.assertAll();
+    }
+
+    @Test(priority = 1)
+    public void assertLogin() {
+        SoftAssertions softly = new SoftAssertions();
+        Header header = new Header(testDriver);
```

Comment:
Please read this very good article https://habr.com/ru/company/jugru/blog/338836/

Make this code according to Page Element pattern

Reply:
Tried to fix :)